pkg/cluster/install.go

```diff
@@ -42,6 +42,7 @@ func (i *manager) AdminUpgrade(ctx context.Context) error {
 		steps.Action(i.upgradeCertificates),
 		steps.Action(i.configureAPIServerCertificate),
 		steps.Action(i.configureIngressCertificate),
+		steps.Action(i.addArchitectureVersion),     // TODO(mj): Remove this once all cluster in production has this field populated
```

Comment:
we can assume that our current clusters are at version 0 and omit this step.  In the 4.5 PR we can write a higher version at some point when we write i.doc